verifier/rest/rest.go

```diff
@@ -6,40 +6,58 @@ import (
 	"encoding/base64"
 	"fmt"
 	"log"
-	"strings"
+	"time"
 
+	"github.com/cenkalti/backoff/v4"
 	sabi "github.com/google/go-sev-guest/abi"
 	"github.com/google/go-sev-guest/proto/sevsnp"
 	tabi "github.com/google/go-tdx-guest/abi"
 	"github.com/google/go-tdx-guest/proto/tdx"
 	"github.com/google/go-tpm-tools/verifier"
 	"github.com/google/go-tpm-tools/verifier/oci"
+	"github.com/googleapis/gax-go/v2"
 
 	v1 "cloud.google.com/go/confidentialcomputing/apiv1"
 	confidentialcomputingpb "cloud.google.com/go/confidentialcomputing/apiv1/confidentialcomputingpb"
 	"google.golang.org/api/iterator"
 	"google.golang.org/api/option"
 	locationpb "google.golang.org/genproto/googleapis/cloud/location"
+	"google.golang.org/grpc/codes"
 )
 
-// BadRegionError indicates that:
-//   - the requested Region cannot be used with this API
-//   - other Regions _can_ be used with this API
-type BadRegionError struct {
-	RequestedRegion  string
-	AvailableRegions []string
-	err              error
+func confComputeBackoffPolicy() backoff.BackOff {
```

Comment:
Seems like this not not needed, it's not referenced anywhere

Reply:
Good catch, removed.